stdlib/public/stubs/Stubs.cpp

```diff
@@ -25,16 +25,18 @@
 // Avoid defining macro max(), min() which conflict with std::max(), std::min()
 #define NOMINMAX
 #include <windows.h>
-#else
-#if !defined(__HAIKU__) && !defined(__wasi__)
+#else // defined(_WIN32)
+#if !defined(__HAIKU__) && !defined(__wasi__) && __has_include(<sys/errno.h>)
```

Comment:
I think that the guards here are unnecessary, `__has_inlcude(<sys/errno.h>)` covers everything.  Is there something Im missing?